defaults/main.yml

```diff
@@ -25,3 +25,5 @@ ntp_restrict:
 ntp_cron_handler_enabled: false
 
 ntp_tinker_panic: false
+
+maxdistance: '3'
```

Comment:
This should probably be something like `ntp_chrony_maxdistance`, and it would need to be documented in the README.